test/integration/puppetlabs/services/jruby/tasks_test.clj

```diff
@@ -58,8 +74,8 @@
                   (str module-name "::" name))})
        tasks))
 
-(deftest ^:integration all-tasks-test
-  (testing "requesting all tasks"
+(deftest ^:integration task-data-test
+  (testing "requesting data about a specific task"
     (let [code-dir (ks/temp-dir)
           conf-dir (ks/temp-dir)
           config (jruby-testutils/jruby-puppet-tk-config
```

Comment:
This is repeating code that was factored out in a function below this deftest form (puppet-tk-config)